piano-hero/mobile/src/main/java/com/novoda/pianohero/GameMvp.java

```diff
@@ -0,0 +1,38 @@
+package com.novoda.pianohero;
+
+interface GameMvp {
+
+    interface Model {
+
+        RoundViewModel startGame();
+
+        RoundViewModel onNotesPlayed(Callback callback, Note... note);
+
+        interface Callback {
+            void onSequenceComplete();
+        }
+    }
+
+    interface View {
+
+        void showRound(RoundViewModel viewModel);
+
+        void showGameComplete();
+
+        void showError();
+
+        void showKeyboard();
```

Comment:
Ha! Actually after writing this, I was thinking what if _keyboard_ is actually part of the domain regardless the delivery mechanism (which is the _view_ implementation). At the time of writing my previous comment, I was thinking `keyboard` as in Android keyboard with letters and numbers on it